.gitignore

```diff
@@ -71,5 +71,7 @@ integreat_cms/xliff/download
 .postgres
 
 # Celery
-celerybeat-schedule.db
 celerybeat-schedule
+celerybeat-schedule.db
```

Comment:
```suggestion
celerybeat-schedule.dat
```

Reply:
`celerybeat-schedule.db` is apparently also a thing 😅